ubports.source_location

```diff
@@ -1,2 +1,2 @@
-https://git.sailfishos.org/mer-core/mobile-broadband-provider-info/-/archive/master/mobile-broadband-provider-info-master.tar.bz2
-mobile-broadband-provider-info_20210212.orig.tar.bz2
+https://github.com/sailfishos/mobile-broadband-provider-info/archive/master.tar.gz
```

Comment:
Due to the way our CI work, we shouldn't pull from a nondeterministic URL. We probably have to pull using a tag. Something like:

```suggestion
https://github.com/sailfishos/mobile-broadband-provider-info/archive/20131125+git80.tar.gz
```

Also, I would prefer that the upstream part of our version actually match the tag, but apparently SailfishOS version are a fork of an older version thus older-versioned tag. We will have to leave it like this for now.

Reply:
But does this mean we cannot automatically pull latest, say every week? I mean that file is changing quite often, manual intervention on it should be reduced. But its ok for now probably